src/services/db/RepoService.ts

```diff
@@ -36,8 +33,27 @@ export default class RepoService extends GitHubService {
     this.gitFileSystemService = gitFileSystemService
   }
 
-  isRepoWhitelisted(repoName: string): boolean {
-    return WHITELISTED_GIT_SERVICE_REPOS.split(",").includes(repoName)
+  isRepoWhitelisted(
+    repoName: string,
+    sessionData: UserWithSiteSessionData
+  ): boolean {
+    if (!sessionData.growthbook) return false
+
+    const ggsWhitelistedRepos: {
+      repos: string[]
+    } = sessionData.growthbook.getFeatureValue("ggs_whitelisted_repos", {
+      repos: [],
+    })
+
+    console.log(
+      `I RECEIVED: ${JSON.stringify(
+        ggsWhitelistedRepos
+      )} to compare with ${repoName}. Returning ${ggsWhitelistedRepos.repos.includes(
+        repoName
+      )}`
+    )
```

Comment:
Will this log print out all whitlisted repo names inside `ggsWhitelistedRepos`? Is it necessary?

Reply:
oops, this was for local debug. have updated the log statement - need this temporarily in case we want to debug on staging/prod